common/lib/xmodule/xmodule/discussion_module.py

```diff
@@ -54,9 +69,28 @@ class DiscussionModule(DiscussionFields, XModule):
     js_module_name = "InlineDiscussion"
 
     def get_html(self):
+        from django.contrib.auth.models import User
+        course = self.get_course()
+        user = None
+        user_service = self.runtime.service(self, 'user')
+        if user_service:
+            user_id = user_service.get_current_user().opt_attrs.get('edx-platform.user_id', None)
+            if user_id:
+                user = User.objects.get(id=user_id)
```

Comment:
Rather than this particular ugly, I'd prefer to have you grab `user_service._django_user`. That'll a) save a query, and b) mean that you don't have to actually import from django directly.